src/compiler/transpilers/testTranspiler.ts

```diff
@@ -27,7 +28,12 @@ export function testTranspile(props: ICompileModuleProps) {
 
   const ext = props.fileName ? path.extname(props.fileName) : '.tsx';
 
-  const ast = parseTypeScript(contents, { jsx: ext !== '.ts' });
+  let ast;
+  if (props.useMeriyah) {
+    ast = parseJavascript(contents, { jsx: ext !== '.ts' });
```

Comment:
shouldn't this be `.js`

Reply:
should be always true ;-) js can contain jsx ;-)